docs/schemas/project-registration-bican/controlled-vocabularies/grant-cv.csv

```diff
@@ -0,0 +1,40 @@
+﻿grant

```

Comment:
There are a few more BICAN grants here that we can add in the future, but I'm not sure if we have decided on CV names for them yet.

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/KATHER~1.BAK/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/KATHER~1.BAK/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<!--table
	{mso-displayed-decimal-separator:"\.";
	mso-displayed-thousand-separator:"\,";}
@page
	{margin:.75in .7in .75in .7in;
	mso-header-margin:.3in;
	mso-footer-margin:.3in;}
tr
	{mso-height-source:auto;}
col
	{mso-width-source:auto;}
br
	{mso-data-placement:same-cell;}
td
	{padding-top:1px;
	padding-right:1px;
	padding-left:1px;
	mso-ignore:padding;
	color:black;
	font-size:11.0pt;
	font-weight:400;
	font-style:normal;
	text-decoration:none;
	font-family:Calibri, sans-serif;
	mso-font-charset:0;
	mso-number-format:General;
	text-align:general;
	vertical-align:bottom;
	border:none;
	mso-background-source:auto;
	mso-pattern:auto;
	mso-protection:locked visible;
	white-space:nowrap;
	mso-rotate:0;}
-->
</head>

<body link="#0563C1" vlink="#954F72">



grant_reference_id | grant_title | grant_identifier | funding_agency_reference_id | awardee_reference_id | report_symbol
-- | -- | -- | -- | -- | --
  | An Atlas of Human Brain Cell Variation | 1UM1MH130966-01 | National Institute of Mental Health | Broad Institute | 10523654
  | Functionally guided adult whole brain cell atlas in human and NHP | 1UM1MH130981-01 | National Institute of Mental Health | Allen Institute | 10523848
  | A Multidisciplinary Center for Developing Human and Non-human Primate   Brain Cell Atlases | 1UM1MH130991-01 | National Institute of Mental Health | University of California, San Francisco | 10523935
  | Center for Multiomic Human Brain Cell Atlas | 1UM1MH130994-01 | National Institute of Mental Health | Salk Institute for Biological Studies | 10523973
  | Spatiotemporal epigenomic and chromosomal architectural cell atlas of   developing human brains | 1U01MH130995-01 | National Institute of Mental Health | University of California, Los Angeles | 10523974
  | Bridging Function, Connectivity, and Transcriptomics of Mouse Cortical   Neurons | 1U01MH130907-01 | National Institute of Mental Health | Allen Institute | 10519964
  | Comprehensive single-cell atlas of the developing mouse brain | 1U01MH130962-01 | National Institute of Mental Health | Harvard University | 10523550
  | A Community Resource for Single Cell Data in the Brain​ | 1U24MH130919-01 | National Institute of Mental Health | Allen Institute | 10521124
  | Scalable Molecular Pipelines for FAIR and Reusable BICAN Molecular Data​ | 1U24MH130968-01 | National Institute of Mental Health | Broad Institute | 10523659
  | An extensible brain knowledge base and toolset spanning modalities for   multi-species data-driven cell types​ | 1U24MH130918-01 | National Institute of Mental Health | Allen Institute | 10520941
  | Engagement and outreach to achieve a FAIR data ecosystem for the BICAN​ | 1U24MH130988-01 | National Institute of Mental Health | University of Texas Health Science Center | 10523908



</body>

</html>

Reply:
Sounds good, I can update these once the CV names are finalized